atd-vzd/schema/ems__incidents.sql

```diff
@@ -0,0 +1,147 @@
+	pcr_patient_age integer NULL,
+	pcr_patient_gender text NULL,
+	pcr_patient_race text NULL,
+	mvc_form_airbag_deployment text NULL,
+	mvc_form_airbag_deployment_status text NULL,
+	mvc_form_collision_indicators text NULL,
+	mvc_form_damage_location text NULL,
+	mvc_form_estimated_speed_kph integer NULL,
+	mvc_form_estimated_speed_mph integer NULL,
+	mvc_form_extrication_comments text NULL,
+	mvc_form_extrication_datetime timestamp NULL,
+	mvc_form_extrication_required_flag integer NULL,
+	mvc_form_patient_injured_flag integer NULL,
+	mvc_form_position_in_vehicle text NULL,
+	mvc_form_safety_devices text NULL,
+	mvc_form_seat_row_number text NULL,
+	mvc_form_vehicle_type text NULL,
+	mvc_form_weather text NULL,
+	pcr_additional_agencies text NULL,
+	pcr_transport_priority text NULL,
+	pcr_patient_acuity_initial text NULL,
+	pcr_patient_acuity_final text NULL,
+	unparsed_apd_incident_numbers text NULL,
+	apd_incident_numbers integer[] NULL,
+	geometry public.geometry(point, 4326) NULL,
+    austin_full_purpose bool default null,
+    location_id text default null,
+    latitude float8 default null,
+    longitude float8 default null,
+    apd_incident_number_1 integer default null,
+    apd_incident_number_2 integer default null,
+    mvc_form_date date default null,
+    mvc_form_time time default null
+);
+
+
+
+CREATE OR REPLACE FUNCTION ems_incidents_trigger()
+  RETURNS trigger AS
+$$
+BEGIN
+  update ems__incidents set
+    austin_full_purpose = (
+      select ST_Contains(jurisdiction.geometry, incidents.geometry)
+      from ems__incidents incidents
+      left join atd_jurisdictions jurisdiction on (jurisdiction.jurisdiction_label = 'AUSTIN FULL PURPOSE')
+      where incidents.id = new.id),
+    location_id = (
+      select locations.location_id
+      from ems__incidents incidents
+      join atd_txdot_locations locations on (incidents.geometry && locations.shape and ST_Contains(locations.shape, incidents.geometry))
+      where incidents.id = new.id),
+    latitude = ST_Y(ems__incidents.geometry),
+    longitude = ST_X(ems__incidents.geometry),
+    apd_incident_number_1 = ems__incidents.apd_incident_numbers[1],
+    apd_incident_number_2 = ems__incidents.apd_incident_numbers[2],
+    mvc_form_date = date(ems__incidents.mvc_form_extrication_datetime),
+    mvc_form_time = ems__incidents.mvc_form_extrication_datetime::time
+    where ems__incidents.id = new.id;
+RETURN NEW;
+END;
+$$
+LANGUAGE 'plpgsql';
+
+create trigger ems_incidents_trigger_insert
+after insert on public.ems__incidents
+for each row execute procedure ems_incidents_trigger();
+
+create trigger ems_incidents_trigger_update
+after update on public.ems__incidents
+for each row 
+WHEN (false 
+  or old.geometry IS DISTINCT FROM new.geometry
+  or old.apd_incident_numbers is distinct from new.apd_incident_numbers
+  or old.mvc_form_extrication_datetime is distinct from new.mvc_form_extrication_datetime)
+execute procedure ems_incidents_trigger();
+
+
+
+-- Column Comment
```

Comment:
Mateo had built out these comments, and I think they are really helpful. This would be a good pattern to carry forward when possible in new tables.